packages/@unimodules/core/ios/UMCore/Services/UMLogManager.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019-present 650 Industries. All rights reserved.
```

Comment:
Inconsistency alert here, please decide whether the copyright is since 2015 or 2019 😅